diff --git a/node_modules/@react-native-firebase/crashlytics/android/build.gradle b/node_modules/@react-native-firebase/crashlytics/android/build.gradle
index b211654..40cc7e5 100644
--- a/node_modules/@react-native-firebase/crashlytics/android/build.gradle
+++ b/node_modules/@react-native-firebase/crashlytics/android/build.gradle
@@ -89,7 +89,7 @@ dependencies {
   implementation "com.google.firebase:firebase-crashlytics-ndk"
 }
 
-ReactNative.shared.applyPackageVersion()
-ReactNative.shared.applyDefaultExcludes()
-ReactNative.module.applyAndroidVersions()
-ReactNative.module.applyReactNativeDependency("api")
+//ReactNative.shared.applyPackageVersion()
+//ReactNative.shared.applyDefaultExcludes()
+//ReactNative.module.applyAndroidVersions()
+//ReactNative.module.applyReactNativeDependency("api")